frontend/package.json

```diff
@@ -39,6 +39,12 @@
     "test-cypress-ceph-headless": "cd packages/ceph-storage-plugin/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --config-file cypress-ceph.json --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
     "test-cypress-kubevirt": "cd packages/kubevirt-plugin/integration-tests-cypress && ../../../node_modules/.bin/cypress open --env openshift=true",
     "test-cypress-kubevirt-headless": "cd packages/kubevirt-plugin/integration-tests-cypress && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-pipelines": "cd packages/pipelines-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-pipelines-headless": "cd packages/pipelines-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
+    "test-cypress-gitops": "cd packages/gitops-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-gitops-headless": "cd packages/gitops-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
+    "test-cypress-knative": "cd packages/knative-plugin/integration-tests && ../../../node_modules/.bin/cypress open --env openshift=true",
+    "test-cypress-knative-headless": "cd packages/knative-plugin/integration-tests && node --max-old-space-size=4096 ../../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless --spec \"features/*/*.feature\";",
```

Comment:
Hi, are any of these supposed to run during CI?

Reply:
Yes Dave, below are the scripts needs to be executed on CI, but I don't want to run them on all pr's
test-cypress-gitops-headless [This should run, only when gitOps label is added to the pr]
test-cypress-knative-headless  [This should run, only when knative label is added to the pr]
I will raise this ci implementation as separate pr